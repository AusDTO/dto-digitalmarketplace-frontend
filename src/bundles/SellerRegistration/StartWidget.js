import React from 'react'

import Start from './components/Start';
import RegisterComponent from '../../RegisterComponent'

export const StartWidget = (props) => <Start />

export default new RegisterComponent({ 'seller-registration-start': StartWidget })
