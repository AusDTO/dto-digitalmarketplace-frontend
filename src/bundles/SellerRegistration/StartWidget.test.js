// FIXME This is a workaround.
// @see https://github.com/facebook/react/issues/7386
jest.mock('react-dom');

import React from 'react';
import {StartWidget} from './StartWidget';
import renderer from 'react-test-renderer';

test('StartWidget renders correctly', () => {
  const component = renderer.create(
    <StartWidget deed="/buyers-guide" signup="/signup" />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
