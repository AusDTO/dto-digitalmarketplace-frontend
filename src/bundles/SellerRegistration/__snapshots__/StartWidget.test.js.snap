exports[`test StartWidget renders correctly 1`] = `
<div>
  <h1>
    Joining the Digital Marketplace
  </h1>
  <p>
    To join the Marketplace you need to:
  </p>
  <ul>
    <li>
      Tell us about your company
    </li>
    <li>
      Describe the digital services you\'ll offer and their rates.
    </li>
    <li>
      Provide evidence of your skills for assessment.
    </li>
    <li>
      Agree to the 
      <a
        href="/buyers-guide">
        terms in the deed
      </a>
      .
    </li>
  </ul>
  <p>
    Registering takes about 
    <b>
      30 mins
    </b>
    . You can save your application and continue it later.
  </p>
  <p>
    Before you start
  </p>
  <ul>
    <li>
      By completing this process you are signing the deed, so be sure the authorised person in the company completes this form.
    </li>
  </ul>
  <p>
    <a
      href="/signup"
      role="button">
      Start Now 
    </a>
  </p>
</div>
`;
